REVIEWING.md

```diff
@@ -0,0 +1,32 @@
+# Reviewing and Merging Pull Requests
+
+This document is a guideline for how core contributors should review and merge
+pull requests (PRs). It is intended to outline the lightweight process that
+we’ll use for now. It’s assumed that we’ll operate on good faith for now in
+situations for which process is not specified.
+
+PRs may only be merged after the following criteria are met:
+
+1. It has been open for at least 1 business day
+1. It has no `veto` label on it
+1. It has been 'LGTM'-ed by 3 different reviewers, each from a different
```

Comment:
I know that we had a bunch of discussion around 3, but I thought that we backed out of that and all decided that 1 non-org LGTM was sufficient for now.